packages/ui/src/components/Transactions/Transaction.tsx

```diff
@@ -57,9 +66,21 @@ const Transaction = ({
         aggregatedData={aggregatedData}
         children={
           (isProposer || possibleSigners.length > 0) && (
-            <div className="buttonWrapper">
+            <TransactionFooterStyled>
+              {!!aggregatedData.callData && (
+                <Linkstyled
+                  className="linkIcon"
+                  onClick={onOpenLink}
```

Comment:
I think it's better to use Link component from Mui library instead of box for the links:
```js
                <Linkstyled
                  className="linkIcon"
                  href={getLink()}
                  target="_blank"
                >
                  See callData
                  <LaunchIcon
                    className="icon"
                    fontSize="small"
                  />
                </Linkstyled>
```

Reply:
sounds good